account_payment_term_partner_holiday/models/res_partner.py

```diff
@@ -16,31 +19,36 @@ class ResPartner(models.Model):
         auto_join=True
     )
 
+    def _get_valid_due_date(self, date):
+        if isinstance(date, str):
+            date = fields.Date.from_string(date)
+        is_date_in_holiday = self.is_date_in_holiday(date)
+        while is_date_in_holiday:
+            date = is_date_in_holiday[1]
```

Comment:
Then this would change according to the comments on the other function:
```suggestion
            date = is_date_in_holiday + relativedelta(1)
```

Reply:
Shouldn't it be?:
```suggestion
            date = is_date_in_holiday[1] + relativedelta(days=1)
```

Reply:
As commented [above](https://github.com/OCA/account-payment/pull/429#discussion_r669620630), if we never use the start date, why do we need to return it and then have to select the specific element? This was imagining we only returned the end date in the other fucntion...

Reply:
> But this is not what I want, as the idea is to have the interval for any possible extension. It's not being used right now, but it may be useful on the future.

Ok, it was only a suggestion. In that case, just follow @pedrobaeza’s suggestion 👍